v3/lint/registration.go

```diff
@@ -386,18 +567,41 @@ var globalRegistry = NewRegistry()
 // RegisterLint must be called once for each lint to be executed. Normally,
 // RegisterLint is called from the Go init() function of a lint implementation.
 //
-// RegsterLint will call l.Lint's Initialize() function as part of the
-// registration process.
-//
 // IMPORTANT: RegisterLint will panic if given a nil lint, or a lint with a nil
 // Lint pointer, or if the lint's Initialize function errors, or if the lint
 // name matches a previously registered lint's name. These conditions all
 // indicate a bug that should be addressed by a developer.
+//
+// @deprecated - use RegisterCertificateLint or RegisterRevocationList instead.
```

Comment:
nit: I think this should only say "use RegisterCertificateLint". All calls to RegisterLint can be directly replaced with RegisterCertificateLint.